go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/go-sql-driver/mysql v1.5.0 // indirect
 	github.com/golangci/golangci-lint v1.31.0 // indirect
 	github.com/google/addlicense v0.0.0-20200622132530-df58acafd6d5 // indirect
+	github.com/jaegertracing/jaeger v1.21.0
```

Comment:
I am very reluctant to add this kind of things just for testing (disclaimer: this is a dependency/test argument rather than a library specific one). Also I don't see the need for this given we have a generic test for GRPC without the need of adding a new library: https://github.com/hypertrace/hypertrace-collector/pull/12/files#diff-dc84efdcda988ef18ddc3ae41e3f99d20f5b172fd77cea30016087f334cd9e70R63. Could you elaborate more on the benefit of this?

Reply:
I can remove the Jaeger test. The dependency on Jaeger will be pulled in anyways since the jaeger receiver is in the standard components.

Reply:
It is alright that dependency will be pulled by a dependency as long as we keep our list of dependencies minimal.